lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/classloader/ClassLoaderSafeConnectorSplitSource.java

```diff
@@ -44,6 +46,14 @@ public ClassLoaderSafeConnectorSplitSource(@ForClassLoaderSafe ConnectorSplitSou
         }
     }
 
+    @Override
+    public Optional<List<Object>> getTableExecuteSplitsInfo()
```

Comment:
This method isn't added in "Add SPI for table procedures" commit

Reply:
Yeah - I wanted to keep all "execute splits info" story separate for now. As I expect some discussion around it. I will restructure later on

Reply:
(just a reminder)